src/glue/jobs/json_to_parquet.py

```diff
@@ -254,6 +254,10 @@ def drop_deleted_healthkit_data(
     try:
         glue_client.get_table(DatabaseName=glue_database, Name=deleted_table_name)
     except glue_client.exceptions.EntityNotFoundException:
+        logger.warn(
```

Comment:
Instead of an try-catch-exception, should we have this throw an error if it's not found since we expect the deleted tables to be found?

Reply:
Yeah, I think that's justified.